README.md

```diff
@@ -84,7 +84,7 @@ It will also set up a process that will watch the local files for changes, and r
 
 See docker-rsync-local-rawls.sh for more configuration options.
 
-If Rawls successfully starts up, you can now access the Rawls Swagger page: https://local.broadinstitute.org:20443/
+If Rawls successfully starts up, you can now access the Rawls Swagger page: http://local.broadinstitute.org:20443/
```

Comment:
Why this change? It can be either https or http.

Reply:
it wasn't working for me with https - I figured maybe google had tightened browser behavior or something.

Reply:
I think it was still starting up, and I happened to try http when it came up 🤦